scripts/benchmarks/benches/base.py

```diff
@@ -16,7 +16,7 @@
 class Benchmark:
     def __init__(self, directory):
         self.directory = directory
-        self.adapter_path = os.path.join(options.ur_dir, 'build', 'lib', f"libur_adapter_{options.ur_adapter_name}.so")
+        self.adapter_path = os.path.join(options.ur_dir, 'lib64', f"libur_adapter_{options.ur_adapter_name}.so")
```

Comment:
It can also be in 'lib' on some systems I think

Reply:
Applied, code made more robust wrt lib dir name.